### PR TITLE
CUSTOM-107 use uni image

### PR DESCRIPTION
**Pull request by nikita-b**
https://jira.percona.com/browse/PMM-0

- [ ] Links to other linked pull requests.

### CODE REVIEW

File: build/pmm-server-docker/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM oraclelinux:7
```
Comment (EvgeniyPatlan) [inline build/pmm-server-docker/Dockerfile]: I suggest using slim image. It can help minimize image size

Comment (nikita-b) [reply build/pmm-server-docker/Dockerfile]: Yes, we'll use slim. I just want to start with regular and then switch to slim to solve fewer problems in one step

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2519-f0394b3
client docker - perconalab/pmm-client-fb:PR-2519-f0394b3
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2519-f0394b3.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2519-f0394b3&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2519-f0394b3.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/9992/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/15568/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2519-c2d9644
client docker - perconalab/pmm-client-fb:PR-2519-c2d9644
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2519-c2d9644.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2519-c2d9644&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2519-c2d9644.tar.gz

File: build/pmm-server-docker/Dockerfile (hunk 1 above)
Comment (nikita-b) [reply build/pmm-server-docker/Dockerfile]: heh, here difference between slip and regular:
```
perconalab/pmm-server-fb                                PR-2519-c2d9644   5d08e5a0a214   18 minutes ago      1.76GB
perconalab/pmm-server-fb                                PR-2519-f0394b3   8e1be47a9e6f   About an hour ago   1.78GB
```

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/9994/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/15571/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2519-1a718f5
client docker - perconalab/pmm-client-fb:PR-2519-1a718f5
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2519-1a718f5.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2519-1a718f5&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2519-1a718f5.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/10033/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/15697/

**Pull request closed without merging**